measure_extinction/extdata.py

```diff
@@ -177,7 +177,7 @@ class ExtData:
         tuples are measurement, uncertainty
 
     waves : dict of key:wavelengths
-        key is BANDS, IUE, IRS, etc.
+        key is BAND, IUE, IRS, etc.
```

Comment:
Why change from `BANDS` to `BAND`?

Reply:
Because that is the actual name of the key, see e.g. line 243:
`red_rel_band = red.data["BAND"].get_band_mag(rel_band)`
For some reason, the comments were just not consistent with the actual key name...

Reply:
Good catch!